src/tools/linter-rules.md

```diff
@@ -30,13 +30,13 @@ To avoid the need to individually select compatible linter rules,
 consider starting with a linter rule set,
 which the following packages provide:
 
-[lints]({{site.pub-pkg}}/lints)
+[lints][]
 : The rule sets that the Dart team encourages using. 
```

Comment:
Consider adding "for Dart command line and server programs. See `flutter_lints` for Flutter programs."